series/distroinfo.go

```diff
@@ -71,10 +71,7 @@ func (d *DistroInfo) Refresh() error {
 	f, err := d.fileSystem.Open(d.path)
 	if err != nil {
 		// On non-Ubuntu systems this file won't exist but that's expected.
-		if errors.Cause(err) == os.ErrNotExist {
-			return nil
-		}
-		return errors.Trace(err)
+		return nil
```

Comment:
This replicates the original code and how it handled the missing file. Although I would have thought an `os.ErrNotExist` would have been passed back in all cases. This obviously isn't the case which is a shame.